doc/hopauth_guide.md

```diff
@@ -74,112 +77,130 @@ Logging in goes through the CILogon service and your institutional single sign o
+By default, a new credential will allow you to authenticate with Hopskotch/Kafka, but will only allow reading data which has been designated as public. To read other topics, or to write data to topics, you must explicitly delegate those capabilities to the credential.
+
+To add permissions to a credential, locate it in the table on the main page and click the corresponding 'Manage' button. This will take you to the management page for that credential. If you would like to delete a credential instead, click the corresponding 'Delete' button.
 
-To add permissions to a credential, locate it in the table on the main page and click the corresponding 'Edit' button. This will take you to the management page for that credential. 
+The management page shows all the avaialble permissions that can be added to a credential, as well as all the permissions currently assigned to that credential, any of which can be removed.
 
 [![The credential management page](hopauth_screenshots/cred_management.png)](hopauth_screenshots/cred_management.png)
 
-This page shows all permissions currently assigned to the credential, any of which can be removed, allows adding additional permissions, and allows the entire credential to be deleted. 
+The reason for configuring permissions per-credential, rather than just having one credential per user which has the same permissions as the user is to make it easy to create compartmentalized credentials for specific purposes. For example, you might create one credential with write permission to a single, critical alert topic which can be placed in an automated data processing pipeline, and then create a separate credential with read access to many topics which you keep on your laptop for doing interactive analysis work.
 
-The reason for configuring permissions per-credential, rather than just having one credential per user which has the same permissions as the user is to make it easy to create compartmentalized credentials for specific purposes. For example, you might create one credential with write permission to a single, critical alert topic which can be placed in an automated data processing pipeline, and then create a separate credential with read access to many topics which you keep on your laptop for doing interactive analysis work. 
+The permissions available to associate with your credentials are shown in the 'Available Permissions' section. These will include all of the permissions available to you because of your group memberships. To add one, simply select it from the list and press the 'Add' button. It should then appear in the table above. The permission is named with the topic to which it applies, other columns of that table refers to the type of access to be allowed and the read/write permissions.
```

Comment:
I would rewrite the last sentence something more like:
"The listing will show the topics to which the permission applies, along with that topic's description, the group through which that permission is conferred, and whether the permission allows reading, writing, or both."